Moli/Core/Screens/Profile/ProfileView.swift

```diff
@@ -209,116 +221,104 @@ struct ProfileView: View {
                         UserThumbnailView(userMinified: profileUserImage(), size: .max)
                     }
                     .overlay(Circle().strokeBorder(Color.primaryDark, lineWidth: 3, antialiased: true))
-                    .offset(x: profileImageSizeXOffset, y: profileImageSizeYOffset)
-                }
+                    .offset(x: 0, y: profileImageSizeYOffset)
+                }.padding(.bottom, 40)
                 
-                VStack(alignment: .leading, spacing: 12) {
-                    HStack(spacing: 4) {
-                        Spacer()
-                        
-                        EditProfileView()
-                            .navigatePush(when: $isShowingEditProfile)
-                        
-                        AsyncButton(action: onPressFollowButton) {
-                            Text(followButtonText())
-                        }
-                        .buttonStyle(SuperButtonStyle(.roundedOutline(type: isFollowing() ? .secondary : .primary), size: .small))
-                    }
-                    
-                    VStack(alignment: .leading) {
+                VStack(alignment: .center, spacing: 16) {
+                    VStack (alignment: .center, spacing: 0) {
                         Text(userName())
-                            .font(.title3)
+                            .font(.headingSmall)
+                            .fontWeight(.bold)
                             .lineLimit(1)
-                        
+
                         Text(userHandle())
-                            .font(.subheadline)
+                            .font(.labelMedium)
                             .lineLimit(1)
-                            .foregroundColor(.gray600)
-                        
+                            .foregroundColor(.gray400)
                     }
-                    
+
+                    if let userInfo = userInfo() {
+                        FriendsAndFollowSection(
+                            userInfo: userInfo,
+                            followers: followers(),
+                            friends: friends(),
+                            following: following()
+                        )
+
+                    }
+
                     if let biography = biography() {
                         if !biography.isEmpty {
```

Comment:
:art: Legacy kod från min sida, här kan man istället skriva
```suggestion
                    if let biography = biography(),  !biography.isEmpty  {
```